release-notes/opensearch-dashboards-reports.release-notes-1.3.0.0.md

```diff
@@ -0,0 +1,18 @@
+### Version 1.3.0.0 Release Notes
+
+Compatible with OpenSearch Version 1.3.0 and OpenSearch Dashboards Version 1.3.x
```

Comment:
Is  this supposed to be 1.x vs 1.3.x?

Reply:
I think it should be `Compatible with OpenSearch and OpenSearch Dashboards Version 1.3.0`

Reply:
yes i think plugin and dashboards patch version need to match, opensearch and dashboards patch version don't have to match

Reply:
I think it should be `Compatible with OpenSearch and OpenSearch Dashboards Version 1.3.0`

Reply:
fixed, thx